_docs/_latest/setup/manual-deployment.md

```diff
@@ -0,0 +1,162 @@
+    gcloud compute firewall-rules create forseti-server-allow-ssh-external --action ALLOW --target-service-accounts <SERVICE_ACCOUNT_EMAIL_ADDRESS> --priority 0 --direction INGRESS --rules tcp:22 --source-ranges 0.0.0.0/0
+    ```
+
+### Configuration
+
+Forseti server configuration and rule files are configured and stored in
+Cloud Storage. For more information, see
+[Configuring Forseti]({% link _docs/latest/howto/configure/configuring-forseti.md %}).
+
+### G Suite Integration
+
+To integrate G Suite, you'll need to enable domain-wide delegation in G Suite.
+For more information, see [Enabling G Suite Google Groups Collection]({% link _docs/latest/configure/gsuite.md %})
+
+## Create a Database
+
+### Create a Cloud SQL instance.
+
+To create a For information about the specifications needed for a Cloud
+SQL instance, see the [Cloud SQL template](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/cloudsql/cloudsql-instance.py).
+
+## Deploy a client VM
+
+### Create a Forseti client service account
+
+Create a client service account by running the following command:
+
+```
+gcloud iam service-accounts create forseti-client-gcp-####### --display-name forseti-client-gcp-#######
+```
+
+Where `#######` is a random alphanumeric unique identifier.
+
+### Assign roles
+
+For information about the roles you need to assign to the Forseti server
+service account, see
+[the client service account]({% link _docs/latest/concepts/service-accounts.html#the-client-service-account %}).
+
+### Create a Forseti client VM Instance
+
+* For information about the specifications needed for a Forseti server
+  VM instance, see the [Compute Engine deployment template](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py)
+* When you create the instance, bind the server service account to the VM instance
+* [Enable computeOS login](https://cloud.google.com/compute/docs/instances/managing-instance-access#enable_oslogin)
+
+### Install the Forseti client
+
+To install the Forseti client, follow the steps below:
+
+1. SSH into the client VM
+    ```bash
+    gcloud compute --project <YOUR_PROJECT> ssh --zone <YOUR_ZONE> <YOUR_FORSETI_CLIENT_NAME>
+    ```
+1. Run the following command to become an Ubuntu user:
+    ```bash
+    sudo su ubuntu
+    ```
+1. `git clone` from the [forseti repo](https://github.com/GoogleCloudPlatform/forseti-security), and check out the [latest release](https://github.com/GoogleCloudPlatform/forseti-security/releases) by their tags.
+    ```bash
+    git clone https://github.com/GoogleCloudPlatform/forseti-security.git
+ 
+    git checkout tags/<tag_number>
+    ```   
+1. Follow the setup instructions in the [startup-script](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/client/forseti-instance-client.py#L93) to install Forseti CLI.
```

Comment:
"to install the Foresti command-line interface (CLI)."

Reply:
Updated